release_scripts/Step2_Offline_PackageSigning.py

```diff
@@ -226,6 +226,128 @@ def getDstPath(pkgName, suffixStr='FileSuffix'):
+         signature = open(sigFile).read()
+      except:
+         print 'Signature file does not exist'
+         continue
+      sig, msg = readSigBlock(signature)
+      addrB58 = verifySignature(sig, msg, 'v1', ord(ADDRBYTE))
+      if addrB58 == signer.split('/')[1]:
+         print 'Signature verified successfully for %s' % signer
+         # Remove "- " PGP message escaping (from RFC 2440)
+         lines = msg.split('\n')
+         for idx, line in enumerate(lines):
+            if line[0:2] == '- ':
+               lines[idx] = line[2:]
+         msg = '\n'.join(lines)
+         
+         # Gitian YAML assert file parsing for hash
+         if pkgInfo['GitianExt'] == '.assert':
+            yaml.add_constructor(u'!omap',
+                    yaml.constructor.SafeConstructor.construct_yaml_omap)
+            msgYaml = yaml.load(msg)
+            msgHashes = msgYaml[0][1].split('\n')[:-1]
+            for msgHash in msgHashes:
+               hashDict[signer].append(msgHash.split('  ')[0])
+
+         # Debian buildinfo file parsing for hash
+         if pkgInfo['GitianExt'] == '.buildinfo':
+            for paragraph in deb822.Deb822.iter_paragraphs(msg):
+               for item in paragraph.items():
+                  if item[0] == 'Checksums-Sha256':
+                     lines = item[1].split('\n')
+                     for line in lines:
+                        line = line.strip()
+                        if line[-3:] == 'deb' and line[-4:-3] == '.':
+                           hashDict[signer].append(line.split(' ')[0])
+
+         # Continue with code common to both assert and buildinfo files
+         try:
+            if hashDict[signer][0] == hashDict[oldSigner][0]:
+               print 'Hash match for signers %s and %s' % (signer, oldSigner)
+               oldSigner = signer
+               sigCnt += weights[i]
+            else:
+               print ('Hash mismatch for signers %s and %s. Signature does'
+                      ' not count as valid.') % (signer, oldSigner)
+         except:
+            # First time, so no oldSigner yet.
+            print ('Did not compare hash for signer %s, because this is the'
+                   ' first signer with a valid signature. Signature is'
+                   ' considered valid and counts towards meeting'
+                   ' threshold.') % signer
+            oldSigner = signer
+            sigCnt += weights[i]
+            continue
+      else:
+         print 'Signature was not successfully verified for signer %s' % signer
+   if sigCnt < threshold:
+      print ''
+      print ('The number of valid signatures necessary to continue was not'
+             ' met. Please run the script again after obtaining enough'
+             ' signatures from the Armory signature repo on GitHub. The'
+             ' signature threshold is %s of %s valid signatures.') % (
+                     threshold, len(signers))
+      exit(1)
+
+   print ''
+   f = open(getSrcPath(pkgName), 'rb')
```

Comment:
Just in case you weren't aware, you can combine this line and the next into "with open(getSrcPath(pkgName) as f:". It's what most Python programmers do. I don't mind if you keep the code in its current form. Just pointing it out. :)